backend/report_submission/storages.py

```diff
@@ -0,0 +1,9 @@
+from django.conf import settings
+from storages.backends.s3boto3 import S3Boto3Storage
+
+
+class S3PrivateStorage(S3Boto3Storage):
```

Comment:
Can we add a comment here explaining that this class override exists to get around the fact that `django-storages` by default references the `AWS_STORAGE_BUCKET_NAME` environment variable. Without this (or something similar), we'd be unable to have multiple S3 buckets (e.g. one public, one private)

Reply:
Added!